tests/test_models.py

```diff
@@ -746,6 +748,19 @@ def test_fusion(
     msg = "Value error, First structural element cannot be LinkerSequence"
     check_validation_error(exc_info, msg)
 
+    with pytest.raises(ValidationError) as exc_info:
```

Comment:
Should there be a test case for multiple errors?

Reply:
messaged on slack! but for documentation purposes I should prob post here too: with the current implementation, I don't think there's currently a way to have multiple messages associated with one ValidationError. And only one ValidationError can be thrown for a fusion at a time. I did write a test case for this initially where I tried to have a fusion start with a linker element and have a transcript segment element that was a 5' partner without an end. In that case, it threw the error that was thrown first. Should we perform multiple validation checks at a given time and append them to a list and throw one ValidationError with all of the associated messages?

Reply:
Yes, I think it would be good to report a single validation error with all of the error messages.

Reply:
updated in latest commit!

Reply:
oh wait, I think this test still doesn't quite test the getting multiple error messages from a pydantic validation error.

Reply:
I can't find a way to do this with the current models, since the validators run at different times, and if I trigger a field validation error for something like assay/causativeEvent, then the validator for for the structural ends won't run since it runs after the pydantic model validation occurs, and only one seems to get caught at a time. I'll add a test for this in `test_tools` so I don't keep spending time trying to get it to work when I don't think it's currently possible

Reply:
nvm, finally came up with an example that works for calling the fusor.fusion function! updated